snapshots/who/2024-04-09/polio_historical.xls.dvc

```diff
@@ -0,0 +1,27 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: WHO Historical Polio Dataset
+    date_published: "2019-12-10"
+
+    # Citation
+    producer: World Health Organization
+    citation_full: |-
+      World Health Organization (2019)
+    attribution_short: WHO
+
+    # Files
+    url_main: https://web.archive.org/web/20200101000000*/http://www.who.int/entity/immunization/monitoring_surveillance/data/incidence_series.xls?ua=1
```

Comment:
Hmm, url_main is supposed to be a valid URL where you can see something about the data. In this case, it downloads a file. Maybe you could include a `description`, where you mention what the dataset contains, and that WHO archived this data, and there you add the link. Up to you, neither solution is optimal.

Reply:
Good point, I will add this to the `description` and then add more general link to `url_main`.